src/PostgREST/App.hs

```diff
@@ -257,8 +269,16 @@ app dbStructure conf apiRequest =
 
       mapSnd f (a, b) = (a, f b)
       readReq = readRequest (configMaxRows conf) (dbRelations dbStructure) (map (mapSnd pdReturnType) $ dbProcs dbStructure) apiRequest
+      fldNames = fieldNames <$> readReq
+      fieldNames :: ReadRequest -> [FieldName]
```

Comment:
I'll move ```fieldNames``` to DbRequestBuilder.hs, I think is related to the module since is used to build ```mutateRequest```.